### PR TITLE
[CIRCLE-24559] Update Java OOM docs to reference cgroups

### PR DESCRIPTION
**Pull request by pete-woods**
# Description
Update the Java docs to reference the upcoming rollout of cgroups information.

# Reasons
https://circleci.atlassian.net/browse/CIRCLE-24559

Because we are planning to rollout out a change to include the cgroups information in Nomad containers, the docs should include updated information to reflect this.

### CODE REVIEW

File: jekyll/_cci2/java-oom.md [hunk 1]
```diff
@@ -146,9 +146,9 @@ to overwrite memory limits set in `JAVA_TOOL_OPTIONS`.
 Unfortunately, debugging Java OOM errors often comes down to finding an `exit
 code 137` in your error output. 
 
-Ensure that your `-Xmxn` maximum size is large enough for your applications to
+Ensure that your `-XX:MaxRAMPercentage` maximum size is large enough for your applications to
```
Comment (glenjamin) [inline jekyll/_cci2/java-oom.md:149]: I think we should still mention Xmx here as well as percentage

File: jekyll/_cci2/java-oom.md [hunk 2]
```diff
@@ -13,15 +13,23 @@ Without any memory limits, the JVM pre-allocates a fraction of
...
-on the machine. This can lead to the JVM seeing a large amount of memory
-being available to it, and trying to use more than is allocated to the
-container.
+on the machine.
 
-This pre-allocation can produce Out of Memory (OOM) errors,
-which are difficult to debug because the error messages lack detail.
+As of [June 03, 2020](https://circleci.com/changelog/#container-cgroup-limits-now-visible-inside-the-docker-executor)
+these limits are visible when using the Docker executor. This means that the recent versions of Java will correctly
+detect the number of CPUs and amount of RAM available to the job.
```
Comment (glenjamin) [inline jekyll/_cci2/java-oom.md:20]: What does the JVM pick as max heap size I’m with cgroups visible and no other flags?

Comment (pete-woods) [reply jekyll/_cci2/java-oom.md:24]: Added the deets!

**Comment by pete-woods** [comment]
> I think we should mention the GC threads thing - as that gives us something to point at if existing builds start failing

Added some copy for this :)

**Pull request merged**